var/spack/repos/builtin/packages/py-nptyping/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyNptyping(PythonPackage):
+    """Python morphology manipulation toolkit"""
+
+    homepage = "https://github.com/ramonhagenaars/nptyping"
+    url = "https://github.com/ramonhagenaars/nptyping/archive/v1.4.0.tar.gz"
+
+    # expand=False because the URL is a wheel
+    version('1.4.0', sha256='dada081b24cd922bbfb0a862e72104d811699fd9f8c9b7bea464fb933dc491a3')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+
+    depends_on('py-typish@1.5.2:', type='run')
```

Comment:
Is this right?

https://github.com/ramonhagenaars/nptyping/blob/v1.4.0/setup.py#L16 wants `>=1.7.0`, and this commit is adding `1.9.1`

Reply:
Yes, this is the latest on pypi. I think adding the later, the better, no ? Potentially, that will make this dependency last longer without requiring any new update.

Reply:
Sorry to jump in, I think the question was about this `typish` 1.5.2 requirement. Shouldn't the requirement be the same as in the `setup.py` linked by Mike?
```
    depends_on('py-typish@1.7.0:', type='run')
```

Reply:
I see, I see. Indeed I had missed the point. Thanks.